src/environment.js

```diff
@@ -0,0 +1,40 @@
+export class Environment {
+  /**
+   * @param {String} environment overwritten environment
+   * @param {Object} base overwrite window to get location
+   */
+  constructor(environment = '', base = window.location.href) {
+    let selectedEnvironment = 'production';
+    if (!environment) {
+      if (base.includes('localhost')) {
+        selectedEnvironment = 'development';
+      }
+      if (base.includes('dev')) {
+        selectedEnvironment = 'development';
+      }
+      if (base.includes('uat')) {
+        selectedEnvironment = 'uat';
```

Comment:
can you add tst/test and stg/staging to the case statement.

Reply:
What are the environment urls?